packages/components/src/Chips/InternalChipDismissible/InternalChipDismissible.tsx

```diff
@@ -3,33 +3,48 @@ import styles from "./InternalChipDismissible.css";
 import { useInternalChipDismissible } from "./hooks";
 import { InternalChipDismissibleInput } from "./InternalChipDismissibleInput";
 import { InternalChipDismissibleProps } from "./InternalChipDismissibleTypes";
-import { ChipDismissible } from "../ChipDismissible";
+import { InternalChip } from "../InternalChip";
+import { InternalChipButton } from "../InternalChipButton";
 
 export function InternalChipDismissible(props: InternalChipDismissibleProps) {
   const {
     availableChipOptions,
     ref: wrapperRef,
     sortedVisibleChipOptions,
-    handleChipClick,
     handleChipAdd,
+    handleChipClick,
+    handleChipKeyDown,
     handleChipRemove,
     handleCustomAdd,
     handleEmptyBackspace,
+    handleWrapperKeyDown,
   } = useInternalChipDismissible(props);
 
   return (
     <div
       ref={wrapperRef}
       className={styles.wrapper}
       data-testid="dismissible-chips"
+      onKeyDown={handleWrapperKeyDown}
     >
       {sortedVisibleChipOptions.map(chip => (
-        <ChipDismissible
+        <InternalChip
           key={chip.value}
           {...chip}
+          onKeyDown={handleChipKeyDown(chip.value)}
           onClick={handleChipClick(chip.value)}
-          onRequestRemove={handleChipRemove(chip)}
-        />
+          suffix={
+            <InternalChipButton
+              icon="remove"
+              {...chip}
+              onClick={handleChipRemove(chip.value)}
+            />
+          }
+        >
+          <span className={styles.visuallyHidden}>
+            Press delete to remove chip
```

Comment:
Discussed with Adam and I don't have an answer beyond changing it to
- “Press backspace on Windows or delete on a Mac to remove Chip” which seems extremely annoying
- do a UserAgent check or some very specific @supports CSS query and change the label based on the response (Like check if os is mac or windows, then change the wording accordingly)

I’m asking in an accessibility-focused channel in the DS slack because I assume we're not the first people to have this problem, but Google did not help me.

(I would prefer to not hang up this PR on this question if it's the only outstanding issue because I think we have some decisions to make)

Reply:
(I’m also not sure the current state of UA-sniffing in browser APIs 😅 )

Reply:
I did get some feedback from a fairly well-known accessibility pro that saying something general like

"Press delete or backspace to remove {label}" would be a reasonable approach

Reply:
I like that. Some classic keep it simple!